src/Components/Artwork/Details.tsx

```diff
@@ -163,13 +163,6 @@ const SaleMessage: React.FC<DetailsProps> = ({
     return <>{highestBid_display || openingBid_display || ""}</>
   }
 
-  if (sale_message?.toLowerCase() === "contact for price") {
-    return <>Price on request</>
-  }
-
-  if (sale_message?.toLowerCase() === "inquire about availability") {
-    return <>{NBSP}</>
-  }
```

Comment:
Removal of this condition will enable the sale message to be displayed here again

<img width="211" alt="Screenshot 2024-06-28 at 13 26 26" src="https://github.com/artsy/force/assets/8002618/bea83abc-13af-46c5-8278-a39ed8d7622c">